docs/dev/build-instructions/build_linux.md

```diff
@@ -5,6 +5,8 @@ version of Mumble, checkout [this file](build_static.md).
 
 ## Dependencies
 
+<details>
```

Comment:
You could auto-open the ubuntu part as a standard, with:
```
<details open>
<summary>...</summary>

</details>  
```

Reply:
Thanks for the suggestion. Implemented.